extensions/tests/unit.tftest.hcl

```diff
@@ -61,41 +77,39 @@ override_data {
 run "aws_ssm_parameter_unit_test" {
   command = plan
 
+  # Configuration
```

Comment:
Ignore these, they are changes in readiness for what I was trying to do before I went down this rabbit hole.